components/form/AddNewItemForm.tsx

```diff
@@ -145,13 +168,17 @@ export default function AddNewItemForm({
               <option value={'Poor'}>Poor</option>
               <option value={'New'}>New</option>
             </select>
-            <p className='error-message'>{errors.condition?.message}</p>
+            {(!conditionValue || errors.condition) && (
+              <p className='error-message'>
+                {errors.condition?.message || 'This field is required'}
+              </p>
+            )}
           </label>
           <label
             htmlFor='item_type'
             className='flex flex-col items-center gap-1 font-light'
           >
-            Categories
+            <span className='flex items-center'>Categories</span>
             <select
               {...register('item_type', { required: 'Required' })}
               className='input-text '
```

Comment:
Watch out for how this will force items to shift around on the page:

![Screenshot 2024-09-20 at 08 28 51](https://github.com/user-attachments/assets/d0347dd6-c40b-4acf-8066-5743dcc9ac27)

Reply:
Thank you! I will make sure to check this.